### PR TITLE
fix: icons, interactionsTests, testIds types exports

### PR DESCRIPTION
**Pull request by SergeyRoyt**
https://monday.monday.com/boards/3532714909/pulses/4633056724

### CODE REVIEW

File: webpack/published-ts-components.js [hunk 1]
```diff
@@ -103,8 +103,6 @@ const publishedTSComponents = {
   useActiveDescendantListFocus: "hooks/useActiveDescendantListFocus",
   useListenFocusTriggers: "hooks/useListenFocusTriggers",
   useSwitch: "hooks/useSwitch",
-  // Don't remove next line
```
Comment (shlomitc) [inline webpack/published-ts-components.js:106]: are you sure? :)

File: package.json [hunk 2]
```diff
@@ -12,14 +12,27 @@
...
+    "*": {
+      "icons": [
+        "./dist/esm/components/Icon/Icons/index.d.ts"
+      ],
+      "interactionsTests": [
+        "./dist/esm/tests/interactions-utils.d.ts"
+      ],
+      "testIds": [
+        "./dist/esm/tests/test-ids-utils.d.ts"
+      ]
```
Comment (shlomitc) [inline package.json:33]: Why are these two needed to be publically exposed APIs?
The `interactionsTests` are mostly wrappers from regular DOM functions and they are very platform specific

File: webpack/published-ts-components.js (hunk 1 above)
Comment (SergeyRoyt) [reply webpack/published-ts-components.js:106]: Pretty sure
We have this marker on the line 87

File: package.json (hunk 2 above)
Comment (SergeyRoyt) [reply package.json:33]: `testIds` are containing `ComponentDefaultTestId` which contains all default data-testid for components and it's being used already in the monolith in cypress tests selectors
image

`interactionsTests` are exporting functions which helps to build storybook interaction tests and this endpoint is being used in the `monday-ui-components` rn
image

Also third one should be `storybookComponents` to export components we are using to build our storybook, but as they are not converted to ts, they are not here

Comment (shlomitc) [reply package.json:33]: ok. I'm not too fond of it as a public API. it creates a coupling between Vibe's internal testing tools with others who consume it, making it harder to refactor. if that's the current state, then fine, but let's try to reduce such dependencies

**Review by shlomitc — APPROVED** [review]

**Pull request merged**